Submitted by:            Chris Punches <chris.punches@silogroup.org>
Date:                    2021-07-28
Initial Package Version: 2.33
Origin:                  Bison 2.3.x is too strict
Description:             Fixes the BISONFLAGS variable in the intl 
                         makefile for glibc 2.33

--- glibc/intl/Makefile	2021-02-01 12:15:33.000000000 -0500
+++ glibc/intl/Makefile	2021-07-28 03:04:22.665138720 -0400
@@ -155,7 +155,7 @@
 
 CPPFLAGS += -D'LOCALEDIR="$(localedir)"' \
 	    -D'LOCALE_ALIAS_PATH="$(localedir)"'
-BISONFLAGS = --yacc --name-prefix=__gettext --output
+BISONFLAGS = --yacc -Wno-error=yacc --name-prefix=__gettext --output
 
 $(inst_localedir)/locale.alias: locale.alias $(+force)
 	$(do-install)
